src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4638,4 +4638,14 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ConstantStringTooLong" xml:space="preserve">
     <value>Length of String constant exceeds current memory limit.  Try splitting the string into multiple constants.</value>
   </data>
+  <data name="ERR_LoadDirectiveOnlyAllowedInScripts" xml:space="preserve">
+    <value>#load is only allowed in scripts</value>
+  </data>
+  <data name="ERR_PPLoadFollowsToken" xml:space="preserve">
+    <value>Cannot use #load after first token in file</value>
+  </data>
+  <data name="CouldNotFindFile" xml:space="preserve">
+    <value>Could not find file.</value>
```

Comment:
Correct (it's a placeholder).

Regarding periods, we're pretty inconsistent, but I like periods.  The full message will be:

```error CS1504: Source file '' could not be opened -- Could not find file.```